cmd/controller/main.go

```diff
@@ -55,6 +55,7 @@ var (
 	imageScanJobImagePullPolicy    = pflag.String("image-scan-job-pull-policy", "IfNotPresent", "Image scan job image pull policy")
 	imageScanInitDelay             = pflag.Duration("image-scan-init-delay", 60*time.Second, "Image scan init delay")
 	imagePrivateRegistryPullSecret = pflag.String("image-private-registry-pull-secret", "", "Image private registry pull secret")
+	imageScanServiceAccount        = pflag.String("image-scan-service-account", "", "Image scan job kubernetes service account. Useful for IAM based private registry auth.")
```

Comment:
Shouldn't we document the usage of this flag in our docs? Second sentence might be interpreted as an attempt to avoid writing documentation, I think its unnecessary.